config/default

```diff
@@ -10,17 +10,23 @@ import tarfile
 assets = os.path.join(media.mypath, "assets")
 
 if not os.path.exists(assets):
-  print "Extracting assets..."
+  print("Extracting assets...")
   try:
     with tarfile.open("{}.tbz2".format(assets), "r:*") as tf:
       tf.extractall(media.mypath)
-  except tarfile.ReadError, e:
+  except tarfile.ReadError as e:
     assert 0, ("Failed to extract assets.  Did you forget to"
       " initialize git lfs (see README.md)?")
 
 for file in os.listdir(os.path.dirname(config)):
   if fnmatch.fnmatch(file, "*.default"):
-    execfile(os.path.join(os.path.dirname(config), file))
+    global_namespace = {
+      "__file__": config,
+      "__name__": "__main__",
+    }
+    local_namespace = locals()
+    with open(os.path.join(os.path.dirname(config), file), 'rb') as f:
+      exec(f.read(), global_namespace, local_namespace)
```

Comment:
Can we use the current default globals and locals parameter behavior?

Reply:
fixed the warning, and changed for exec